apps/web/src/app/(signing)/sign/[token]/form.tsx

```diff
@@ -103,6 +111,25 @@ export const SigningForm = ({ document, recipient, fields }: SigningFormProps) =
                 />
               </div>
 
+              <div>
+                <Label htmlFor="date-format">Date Format</Label>
+
+                <Select
+                  onValueChange={(value) => {
+                    setDateFormat(value);
+                  }}
+                  defaultValue={dateFormat}
+                >
+                  <SelectTrigger className="bg-background mt-2">
+                    <SelectValue />
+                  </SelectTrigger>
+                  <SelectContent>
+                    <SelectItem value="yyyy-MM-dd hh:mm a">YYYY-MM-DD</SelectItem>
+                    <SelectItem value="dd/MM/yyyy hh:mm a">DD/MM/YYYY</SelectItem>
+                  </SelectContent>
```

Comment:
This seems to be a great addition, thank you @Hallidayo. Would really love to know your thoughts on adding another date format here, that is `MM/DD/YYYY` format as well

Reply:
If we're here, I'd say it would be a good addition to also add the `MM/DD/YYYY` format suggested by @adithyaakrishna.

Reply:
Hi @adithyaakrishna & @catalinpit

I've moved the static items into a constants file to manage this going forward. This constants file also has `MM/DD/YYYY` format in now:

<img width="377" alt="Screenshot 2023-10-17 at 19 11 23" src="https://github.com/documenso/documenso/assets/22655069/9da5597b-70d8-4195-aca7-665fe53218ce">

Reply:
Awesome, this is super cool! Thanks @Hallidayo 🎉